docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -36,94 +36,59 @@ After you have scaffolded a LoopBack4 application, you’ll get a default endpoi
 
 ## Step 1: Scaffold LoopBack 4 app
 
-Run `lb4 app` command.
+Run the `lb4 app` command, and specify all the values provided below.
 
 ```
 $ lb4 app
-? Project name: deploy-test
-? Project description: loopback example
-? Project root directory: deploy-test
-? Application class name: DeployTestApplication
-? Select features to enable in the project Enable tslint, Enable prettier, Enable mocha, Enable loopback
-Build, Enable vscode, Enable repositories, Enable services
-...
-Application deploy-test was created in deploy-test.
-Next steps:
-$ cd deploy-test
-$ npm start
+? Project name: lb4-simple-web-app
+? Project description: lb4-simple-web-app
+? Project root directory: lb4-simple-web-app
+? Application class name: Lb4SimpleWebAppApplication
+? Select features to enable in the project (Press <space> to select, <a> to toggle all, <i> to invert selection)
+ ◉ Enable tslint: add a linter with pre-configured lint rules
+ ◉ Enable prettier: install prettier to format code conforming to rules
+ ◉ Enable mocha: install mocha to run tests
+ ◉ Enable loopbackBuild: use @loopback/build helpers (e.g. lb-tslint)
+ ◉ Enable vscode: add VSCode config files
+ ◉ Enable docker: include Dockerfile and .dockerignore
+ ◉ Enable repositories: include repository imports and RepositoryMixin
+ ◉ Enable services: include service-proxy imports and ServiceMixin
+(Move up and down to reveal more choices)
 ```
 
-Navigate to the main directory of your project
+The `lb4-simple-web-app` project is created.
 
-```
-cd deploy-test
-```
-
-Let's **not** run `npm start` yet, we have some changes to make to certain
-files.
-
-## Step 2: Modify deploy-test/index.js
-
-{% include tip.html content="It is important for the host to be <b>'0.0.0.0'</b> and the port to be consistent in
-several places. We will use a port of <b>8080</b>." %}
+Navigate to the main directory of the project
 
 ```
-  const config = {
-    rest: {
-      port: 8080,
-      host: '0.0.0.0',
-      openApiSpec: {
-        // useful when used with OASGraph to locate your application
-        setServersFromRequest: true,
-      },
-    },
-  };
+cd lb4-simple-web-app
 ```
 
-## Step 3: Add .dockerignore file
-
-At project root, create a file called `.dockerignore` with the following
-content:
-
-```
-node_modules
-npm-debug.log
-```
-
-## Step 4: Build the application
-
-In a command window, navigate the main directory of your project and type :
-
-```
-npm run build
-```
-
-Make sure there are no errors before continuing to the next step.
-
-## Step 5: Run the application locally
+## Step 2: Run the application locally
 
 In a command window in the main directory of your project, type:
 
 ```
 npm start
 ```
 
-The server should start up successfully and display
+The application will build, and then the server should start up successfully and
+display
 
 ```
-Server is running at http://127.0.0.1:8080
-Try http://127.0.0.1:8080/ping
+Server is running at http://[::1]:3000
+Try http://[::1]:3000/ping
```

Comment:
Should we perhaps modify our project template to use IPv4 address `0.0.0.0` by default, or even `127.0.0.1`, so that we can keep using IPv4 URLs that most people are more familiar with? IDK.

Reply:
@raymondfeng , @dhmlau ,@biniam, @nabdelgadir  ^

Reply:
btw. Biniam's github handle is @b-admike.

Reply:
@raymondfeng , what are your thoughts on @bajtos 's comment?